### ISSUE TITLE
lib folder is missing in 4.2.3

### ISSUE THREAD

**Issue by oferh** [angular-redux/ng2-redux#286]
Version `4.2.3` doesn't include the `lib` folder.
I'm getting the following error after upgrading:

```
ModuleNotFoundError: Module not found: Error: Can't resolve 'ng2-redux' in '/app/path`
```

Rolling back to `4.2.2`

**Comment by 11mb** [comment]
It seems to me that the rollback for #284 has failed

**Comment by SethDavenport** [comment]
Reproduced. Investigating.

**Comment by SethDavenport** [comment]
Fixed in 4.2.4.

I apologize for the inconvenience. Forgot to run `npm run build` before `npm publish`. I've added a `prepublish` script to stop this from happening again.  4.2.4 restores things to the same state as 4.2.2.

The fix for #284 represents a breaking API change and will therefore be pursued in a new major version.
